app/views/catalog/_search_form_advanced_modal.html.erb

```diff
@@ -14,9 +14,11 @@
       </div>
       <div class="modal-body">
         <%= form_tag search_action_url, method: :get, class: 'search-query-form advanced-search pr-0 flex-fill', role: 'search' do %>
-          <%= render_hash_as_hidden_fields(search_state.params_for_search.except(:q, :search_field, :qt, :page, :utf8).merge(f: (search_state.params_for_search[:f] || {}).except(:collection_sim))) %>
+          <%= render_hash_as_hidden_fields(search_state.params_for_search.except(:q, :search_field, :qt, :page, :utf8).merge(f: (search_state.params_for_search[:f] || {}).except(:level_sim, :repository_sim, :collection_sim))) %>
           <%# DUL CUSTOMIZATION: default to group by collection %>
           <%= hidden_field_tag :group, 'true', {:id => "group_advanced"} %>
```

Comment:
@gkostin1966 the `group=true` hidden field is still present but is getting lost in the query.

Reply:
That is the advanced search form a.k.a. dialog box.  Does the syntax look right to you?